### PR TITLE
Define axisdir with points explictly

### LINKED ISSUE #1623: Add  an optional additional agrugment to assign_lumped_rlc_to_sheet.
### Description of the feature

In method , assign_lumped_rlc_to_sheet, allow the user to enter a start and stop point, then the user doesn't have to give you the axis.  

In another words, The use can provide the start and stop for props["CurrentLine"], instead of passing axisdir.

`props["CurrentLine"] = OrderedDict({"Start": start, "End": stop})`

https://aedt.docs.pyansys.com/API/_autosummary/pyaedt.Hfss.assign_lumped_rlc_to_sheet.html

### Steps for implementing the feature

Allow user a choice of passing either asixdir or start/stop 3-d points.

### Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**
Fix #1623 

Sometimes is not clear how the integration line is defined, mainly, when the sheet is not parallel to the CS. I think these methods should have the option of defining explicitly the coordinates of the integration lines.

### CODE REVIEW

File: pyaedt/hfss.py [hunk 1]
```diff
@@ -3485,9 +3517,10 @@ def assign_lumped_rlc_to_sheet(
         ----------
         sheet_name : str
             Name of the sheet to apply the boundary to.
-        axisdir : int or :class:`pyaedt.application.Analysis.Analysis.AxisDir`, optional
-            Position of the port. It should be one of the values for ``Application.AxisDir``,
-            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``.
+        axisdir : int, :class:`pyaedt.application.Analysis.Analysis.AxisDir` or list, optional
+            Direction of the integration line. It should be one of the values for ``Application.AxisDir``,
+            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``. It also allows give the list
```
Comment (MaxJPRey) [inline pyaedt/hfss.py:3522]:
```suggestion
            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``. It also accepts the list
```

File: pyaedt/hfss.py [hunk 2]
```diff
@@ -3353,9 +3375,10 @@ def assign_current_source_to_sheet(self, sheet_name, axisdir=0, sourcename=None)
...
```
Comment (MaxJPRey) [inline pyaedt/hfss.py:3380]:
```suggestion
            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``. It also accepts the list
```

File: pyaedt/hfss.py [hunk 3]
```diff
@@ -3310,9 +3321,10 @@ def assign_voltage_source_to_sheet(self, sheet_name, axisdir=0, sourcename=None)
...
```
Comment (MaxJPRey) [inline pyaedt/hfss.py:3326]:
```suggestion
            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``. It also accepts the list
```

File: pyaedt/hfss.py [hunk 4]
```diff
@@ -3209,9 +3209,10 @@ def create_lumped_port_to_sheet(
         ----------
         sheet_name : str
             Name of the sheet.
-        axisdir : int or :class:`pyaedt.application.Analysis.Analysis.AxisDir`, optional
-            Position of the port. It should be one of the values for ``Application.AxisDir``,
-            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``.
+        axisdir : int, :class:`pyaedt.application.Analysis.Analysis.AxisDir` or list, optional
+            Direction of the integration line. It should be one of the values for ``Application.AxisDir``,
+            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``. It also allows give the list
```
Comment (MaxJPRey) [inline pyaedt/hfss.py:3214]:
```suggestion
            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``. It also accepts the list
```

**Comment by Samuelopez-ansys** [comment]
@MaxJPRey Why is Codevoc failing if we didn't touch Cablemodelling.py?

@gmalinve and I will fix it, but we don't know why at the beginning of this PR, we had to improve the coverage of this class.

File: pyaedt/modules/CableModeling.py [hunk 5]
```diff
@@ -225,7 +225,9 @@ def create_cable(self):
                         ["NAME:TwistedPairAttribs", "Name:=", self.cable_name],
                     )
                 else:
-                    self.logger.error("is_lay_length_specified value not valid. Value must be either True or False.")
+                    self._app.logger.error(
+                        "is_lay_length_specified value not valid. Value must be either True or False."
```
Comment (PipKat) [inline pyaedt/modules/CableModeling.py:229]:
```suggestion
                        "Value for lay_length_specified is invalid. Value must be either True or False."
```

**Comment by MaxJPRey** [comment]
> @MaxJPRey Why is Codevoc failing if we didn't touch Cablemodelling.py?
> 
> @gmalinve and I will fix it, but we don't know why at the beginning of this PR, we had to improve the coverage of this class.

Don't worry about it. It is an issue with Codecov. It happens to other PR.

**Comment by priti-ashvin-shah-ibm** [comment]
Thanks for creating this pull request.  I got it to work for my code when I used this branch.

**Comment by Samuelopez-ansys** [comment]
@MaxJPRey Please could you check why Codacy is taking so long? 

Please approva the PR, this PR will increment the coverage in Cable Modelling class as well.

**Review by gmalinve — APPROVED** [review]

File: pyaedt/hfss.py [hunk 6]
```diff
@@ -3209,9 +3209,10 @@ def create_lumped_port_to_sheet(
         ----------
         sheet_name : str
             Name of the sheet.
-        axisdir : int or :class:`pyaedt.application.Analysis.Analysis.AxisDir`, optional
-            Position of the port. It should be one of the values for ``Application.AxisDir``,
-            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``.
+        axisdir : int, :class:`pyaedt.application.Analysis.Analysis.AxisDir` or list, optional
+            Direction of the integration line. It should be one of the values for ``Application.AxisDir``,
+            which are: ``XNeg``, ``YNeg``, ``ZNeg``, ``XPos``, ``YPos``, and ``ZPos``. It also accepts the list
+            of the start point and end point with the format [[xstart, ystart, zstart], [xend, yend, zend]]
```
Comment (Samuelopez-ansys) [inline pyaedt/hfss.py:3215]:
```suggestion
            of the start point and end point with the format [[xstart, ystart, zstart], [xend, yend, zend]].
```

**Pull request merged**